sdk/cosmos/README.md

```diff
@@ -1,296 +0,0 @@
-﻿# Azure CosmosDB Client Library for Java
```

Comment:
FWIW we still recommend having a root level README file that describes the set of packages in the folder. Something like https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/README.md

Reply:
Sure, we can add it. That would be helpful to the users.

Reply:
Probably not as part of this PR, once we improve the content of other READMEs, may be then.